Assets/Scripts/Game/Enemy/EnemyMissile.cs

```diff
@@ -18,19 +23,23 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        
     }
     public async UniTaskVoid Movement()
     {
-        foreach (var enemyPath in enemyPaths)
+        if(enemyPaths != null){
+            paths = enemyPaths.SetEnemyPaths();
+            
+        }
+        
+        for(int i = 0; i < paths.Length;i++)
         {
         transform.DOLocalPath(
-        path     : enemyPath.GetWayPoints(), //移動するポイント
-        duration : enemyPath.GetMoveTime(), //移動時間
+        path     : paths[i].GetWayPoints(), //移動するポイント
```

Comment:
enemyPathのクラスでプロパティを使えば
```suggestion
        path     : paths[i].WayPoints, //移動するポイント
```
みたいにメソッド経由にしつつメンバ変数みたいにかけたりして冗長性が減ったり

Reply:
他の類似した点も同様

Reply:
(でもenemyPath自体がメソッド使って前から実装されていたのか)